neptune/new/handler.py

```diff
@@ -97,6 +174,33 @@ def log(self,
             timestamp: Optional[float] = None,
             wait: bool = False,
             **kwargs) -> None:
+        """Logs the provided value or a collection of values.
+
+        Available for following field types (`Field types docs page`_):
```

Comment:
It's available for FloatSeries, StringSeries & FileSeries

Reply:
Fakt, copy paste - i tutaj nie przeedytowałem...